### ISSUE TITLE
there is no menu i can not find it

### ISSUE THREAD

**Issue by youssef122312** [TomSchimansky/CustomTkinter#820]
i have to use tkinter menu farthermore it is not warking 

self.menu = Menu(master=root, tearoff=False)
        self.menu.add_command(label="Copy", command=self.copy)
        self.menu.add_command(label="Paste", command=self.paste)
        self.menu.add_command(label="Cut", command=self.cut)
        self.menu.add_separator()
        self.menu.add_command(label="Select all", command=self.select_all)
        self.menu.add_separator()
        self.menu.add_command(label="About", command=self.about)
        root.bind("<Button - 3>", self.mypopup)
           def mypopup(self, e):
        self.menu.tk_popup(e.x_root, e.y_root)
    
    def copy(self):
        self.ent.event_generate("<<Copy>>")
    
    def paste(self):
        self.ent.event_generate("<<Paste>>")
    
    def cut(self):
        self.ent.event_generate("<<Cut>>")
        
    def select_all(self):
        self.ent.event_generate("<<SelectAll>>")
    
    def about(self):
        pass

**Comment by TomSchimansky** [comment]
What do you mean by it is not working?

**Comment by jl-houss** [comment]
> i have to use tkinter menu farthermore it is not working

yeah i've researched and the Menu widget isn't implemented in the customtkinter and some others widgets too

**Comment by youssef122312** [comment]
it is not working because it is not exist😂😂😂

**Comment by TomSchimansky** [comment]
Closing this as it looks like a general usage question rather than a bug in CustomTkinter. The best places for usage help are the [Discussions](https://github.com/TomSchimansky/CustomTkinter/discussions) tab, the [Wiki](https://github.com/TomSchimansky/CustomTkinter/wiki), or Stack Overflow. If you can reproduce an actual bug in CustomTkinter with a minimal example, please comment and it'll be reopened.
